c/iotdm_client/lwm2m_objects/device_object.h

```diff
@@ -17,6 +17,13 @@
 
 #include "iothub_client.h"
 
+#ifdef __MACH__
```

Comment:
Interesting.  I may have just fixed this while removing warnings under Linux.  There was some #include funniness that should have caused build failures but our build doesn’t require function prototypes for some reason.

From: Damon Barry [mailto:notifications@github.com]
Sent: Monday, May 9, 2016 3:39 PM
To: Azure/azure-iot-sdks <azure-iot-sdks@noreply.github.com>
Cc: Bert Kleewein <bertk@microsoft.com>; Comment <comment@noreply.github.com>
Subject: Re: [Azure/azure-iot-sdks] Osxbuild (#547)


In c/iotdm_client/lwm2m_objects/device_object.h<https://github.com/Azure/azure-iot-sdks/pull/547#discussion_r62585291>:

> @@ -17,6 +17,13 @@

>

>  #include "iothub_client.h"

>

> +#ifdef __MACH__

Cameron added these to get OSX build working. Not sure why they aren't required in Linux/Windows build? There are defined structs/functions in these headers which are used in this file...we need to investigate why the build doesn't fail in Linux/Windows.

—
You are receiving this because you commented.
Reply to this email directly or view it on GitHub<https://github.com/Azure/azure-iot-sdks/pull/547/files/040f5485101cfaa5907b9d84aff3432dadff1bde#r62585291>